<?xml version="1.0"?>
<launch> 


  <!-- ArUco mapping -->
  <node pkg="aruco_mapping" type="aruco_mapping" name="aruco_mapping">
    <remap from="/image_raw" to="/camera/color/image_raw"/>

    <param name="calibration_file" type="string" value="$(find aruco_mapping)/data/R200.ini" /> 
    <param name="scenario_file" type="string" value="$(find aruco_mapping)/data/scenario_SIMAC.map" /> 
    <param name="num_of_markers" type="int" value="150" />
    <param name="marker_size" type="double" value="0.11"/>
    <param name="space_type" type="string" value="plane" />
    <param name="roi_allowed" type="bool" value="false" /> 
    <param name="roi_x" type="int" value="0" /> 
    <param name="roi_y" type="int" value="0" /> 
    <param name="roi_width" type="int" value="640" /> 
    <param name="roi_height" type="int" value="480" />  

  </node> 
  
</launch>
